tests/spark/generated/rflx-derivation-message.adb

```diff
@@ -56,6 +56,14 @@ is
       Reset (Ctx, Ctx.First, RFLX_Types.To_Last_Bit_Index (RFLX_Types.Length (RFLX_Types.To_Index (Ctx.First)) + Length - 1));
    end Write;
 
+   function Size (Ctx : Context) return RFLX_Types.Bit_Length is
+     ((if
+          Ctx.Message_Last = Ctx.First - 1
+       then
+          0
+       else
+          Ctx.Message_Last - Ctx.First + 1));
+
```

Comment:
IIRC we use this layout only with complex condition (i.e. ones that contain disjunctions/conjunctions). In this case the expressions should come after `if`/`then`/`else` respectively. Would it be complicated to change that in the generator?

Reply:
There could be also complex expressions after `then` or `else`, where this layout would be favorable. It should be possible to use this layout only if a certain line length would be exceeded when putting everything on one line. Do you think it would make sense to decide the layout in this way?

Reply:
I think that would work well. In any case it would be better than most code generators ;)

Reply:
Changed in cc1982d.